client/src/pages/login.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import {Link } from 'react-router-dom';
+import { makeStyles, Container, Typography, TextField, Button, CssBaseline, Divider, FormControlLabel, Checkbox } from '@material-ui/core';
+
+//Comment for observing testing
+
+const useStyles = makeStyles( theme => ({
+    containerTweaks: {
+        marginTop: theme.spacing(4),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        fontFamily: '"ProximaNova"', //Not sure how to change font
```

Comment:
If it's not one of the default fonts, you need to add it first to make it available. Easiest way is to use a cdn and add it to public/index.html, for example:
<link href="https://fonts.googleapis.com/css?family=Titillium+Web:300,400,700" rel="stylesheet">

Look for a CDN for proximaNova, if you can't find it, follow the self-hosting tutorial here:
https://scotch.io/@micwanyoike/how-to-add-fonts-to-a-react-project

Reply:
Also, if you want to use that font in every typography component, you would do that on the theme:
```
typography: {
    fontFamily: "ProximaNova"
  },
```